packages/wrangler/src/dev/start-server.ts

```diff
@@ -296,8 +295,11 @@ async function runEsbuild({
 	};
 }
 
-export async function startLocalServer(props: LocalProps) {
-	if (!props.bundle || !props.format) return Promise.resolve({ stop() {} });
+export async function startLocalServer(
+	props: LocalProps
+): Promise<{ stop: () => Promise<void> }> {
+	if (!props.bundle || !props.format)
+		return Promise.resolve({ async stop() {} });
```

Comment:
This is inside an async function, so can just return the object:
```suggestion
		return { async stop() {} };
```